blaze/expr/strings.py

```diff
@@ -54,7 +55,14 @@ def isstring(ds):
     return isinstance(getattr(measure, 'ty', measure), String)
 
 
+_add, _radd = _mkbin('add', Concat)
+_mod, _rmod = _mkbin('mod', Interp)
+_mul, _rmul = _mkbin('mul', Repeat)
```

Comment:
nice. this should allow backends to use this syntax even if they don't know anything about it